### ISSUE TITLE
Consider adding a 'LGTM' badge to the readme

### ISSUE THREAD

**Issue by Siilwyn** [tgriesser/knex#2731]
With more than 10K line of JS code I think it would be nice to show the LGTM 'A' grade in the readme. Details about the grade can be found in the [LGTM grading docs](https://lgtm.com/help/lgtm/project-scoring-grading-faqs) and Knex project details here: https://lgtm.com/projects/g/tgriesser/knex/

The shields.io service supports LGTM badges.

Thoughts?

**Comment by sjvs** [comment]
I think that's a fantastic idea, but I may well be biased :wink:. https://github.com/tgriesser/knex/pull/2755

Also: I think automatic code review for pull requests is a great way to keep the code quality as great as it is! Here's an example of how Google's AMPHTML team and community use it: https://github.com/ampproject/amphtml/pull/13060. You can enable it for Knex here: https://lgtm.com/projects/g/tgriesser/knex/ci/

(full disclosure: I'm on the LGTM.com team)

**Comment by Siilwyn** [comment]
Hehe, nice @sjvs. Off-topic: how did you find this issue, do you somehow watch for GitHub issues containing LGTM keywords?

**Comment by sjvs** [comment]
I just woke up in the middle of the night from a dream that told me about the Knex project. In the dream, some guy urged me to go and have a look. And it looked good to me :smile: 

But more seriously: we indeed use a service that flags up mentions of LGTM. Turns out that it's quite a popular phrase though :wink:. There are a few such monitoring services available out there, but I'm afraid I haven't actually found one that (1) works well and (2) is reasonably priced.
